### DISCUSSION TITLE
backrest auto tagging

### DISCUSSION THREAD

**Discussion by kinghat** [garethgeorge/backrest#265] [Q&A] [answered]
is backrest auto tagging the snap shot with the plans name: `plan:plan-name` for internal backrest purposes or as a courtesy? if its the latter it might be better to just have an already filled in backup flag `--tag plan:plan-name` when the user is creating a new plan.

**Comment by garethgeorge** [comment, marked as answer]
These are used by backrest to properly re-import your backups on a new system such that they will still be associated with the plan that created them (e.g. if you backup your config somewhere and restore it onto a new host).

Backrest will import snapshots without a plan tag as "unassociated".

**Comment by noelhibbard** [comment]
I just switched from raw restic to backrest. Should I add the plan and created-by tags to my imported snapshots so that they get forgotten according to the retention policy I've created in my new backup plan? I want to hold onto my old snapshots for a while but don't want them hanging on for eternity in my AWS S3 bucket. Would it be better to tag these old snapshots with a unique tag and then make my own cronjob outside of backrest to forget them according to a retention policy?

**Reply by garethgeorge** [reply]
Right — if you add the plan and also created-by tags to match other snapshots they’ll be managed by backrest.
